src/app/menubar/components/menubar-container/menubar-container.component.html

```diff
@@ -1,5 +1,5 @@
 <header>
-  <img src="../../../../assets/images/GO-logo-magenta.png" alt="GO logo" routerLink="/">
+  <img src="../../../../assets/images/GO-logo-magenta.png" alt="GO logo" routerLink="/projecten">
```

Comment:
kan vervangen worden door assets/images/go-logo-magenta.png
is dan niet afhankelijk van in welke map de link gelegd wordt